casepro/cases/tests.py

```diff
@@ -1048,6 +1048,27 @@ def test_create_and_read(self):
         response = self.url_get('unicef', read_url)
         self.assertEqual(response.status_code, 302)
 
+    @override_settings(CELERY_ALWAYS_EAGER=True, CELERY_EAGER_PROPAGATES_EXCEPTIONS=True, BROKER_BACKEND='memory')
+    def test_create_with_no_initial_message(self):
+        '''When a case is exported with initial_message=None, the field should be a blank string.'''
```

Comment:
Addressed in https://github.com/praekelt/casepro/pull/94/commits/8874400907d463cc2f731f2a8c3ee3ab0006c68c